src/Models/DomainRouter.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Cyberfusion\ClusterApi\Models;
+
+use Cyberfusion\ClusterApi\Support\Arr;
+use Cyberfusion\ClusterApi\Contracts\Model;
+
+class DomainRouter extends ClusterModel implements Model
+{
+    private string $domain;
+    private bool $forceSsl = true;
+    private ?int $virtualHostId = null;
+    private ?int $urlRedirectId = null;
+    private ?int $nodeId = null;
+    private ?int $id = null;
+    private ?int $clusterId = null;
+    private ?string $createdAt = null;
+    private ?string $updatedAt = null;
+
+    public function getDomain(): string
+    {
+        return $this->domain;
+    }
+
+    public function setDomain(string $domain): DomainRouter
+    {
+        $this->domain = $domain;
+
+        return $this;
+    }
+
+    public function isForceSsl(): bool
+    {
+        return $this->forceSsl;
+    }
+
+    public function setForceSsl(bool $forceSsl): DomainRouter
+    {
+        $this->forceSsl = $forceSsl;
+
+        return $this;
+    }
+
+    public function getVirtualHostId(): ?int
+    {
+        return $this->virtualHostId;
+    }
+
+    public function setVirtualHostId(?int $virtualHostId): DomainRouter
+    {
+        $this->virtualHostId = $virtualHostId;
+
+        return $this;
+    }
+
+    public function getUrlRedirectId(): ?int
+    {
+        return $this->urlRedirectId;
+    }
+
+    public function setUrlRedirectId(?int $urlRedirectId): DomainRouter
+    {
+        $this->urlRedirectId = $urlRedirectId;
+
+        return $this;
+    }
+
+    public function getNodeId(): ?int
+    {
+        return $this->nodeId;
+    }
+
+    public function setNodeId(?int $nodeId): DomainRouter
+    {
+        $this->nodeId = $nodeId;
+
+        return $this;
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function setId(?int $id): DomainRouter
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    public function getClusterId(): ?int
```

Comment:
```suggestion
    public function getClusterId(): int
```

Reply:
Done in 33fa6ea2939fe757da73a24d061282bf0a67e4f1